src/bpmncwpverify/state.py

```diff
@@ -1,37 +1,30 @@
-from antlr4 import CommonTokenStream, InputStream
+from antlr4 import CommonTokenStream, InputStream, ParseTreeWalker
```

Comment:
All of the symbol table code looks clean to me. I am curious if you think this file is too large and if we should split the functionality into different files

Reply:
Are you referring to the `ParseTreeWalker` or `state.py`. If it's the `ParseTreeWalker`, then that is an ANTLR generated file that we effectively use and don't look at. The `state.py` file is only 206 LOC.